media-video/qmplay2/qmplay2-25.06.27.recipe

```diff
@@ -26,13 +26,14 @@ REQUIRES="
 	haiku${secondaryArchSuffix}
 	qt6_tools${secondaryArchSuffix}
 	cmd:yt_dlp
-#	lib:libass$secondaryArchSuffix
+	lib:libass$secondaryArchSuffix
```

Comment:
Checking the library and the modules show no sign of the enabled/new dependencies?

Reply:
For CMake, LIBASS=OFF for now - but included in the build. Some integration testing was needed on harfbuzz...